### ISSUE TITLE
Using it without a keyboard not working

### ISSUE THREAD

**Issue by injainja** [GoogleForCreators/web-stories-wp#3479]
## Bug Description

maybe I am missing something there but I cannot find way to use the plug-in on my tablet without a keyboard. I cannot make text editable.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by LuckynaSan** [comment]
@injainja Sorry to hear you're experiencing an issue. Hopefully, we can resolve it. 
- What type of tablet device are you on?
- What browser (and version) are you using?

**Comment by injainja** [comment]
I am having this issue with all mobile devices I own and all browsers I use. E.g. LGV30 with chrome:
I tap into a text box which selects the box itself. But there is no way to make the text editable - neither via a second, nor a double tap.

**Comment by LuckynaSan** [comment]
Hello @injainja! Thank you for your patience in waiting for a response. Mobile phone support is out of scope, but tablet support is on the roadmap. We are hoping to work on this, so please feel free to reference GitHub issue #1686 for the latest information.

**Comment by injainja** [comment]
Thank you. Tablet will suffice 🤗
